packages/mix/lib/src/specs/icon/icon_widget.dart

```diff
@@ -50,28 +51,32 @@ class IconSpecWidget extends StatelessWidget {
     this.semanticLabel,
     super.key,
     this.textDirection,
-    this.modifierOrder = const [],
+    this.orderOfModifiers = const [],
   });
 
   final IconData? icon;
   final IconSpec? spec;
```

Comment:
For the Widget spec, all of them follow the same basic structure, a spec attribute with some `Spec` and an attribute called orderOfModifiers. What if we create an `abstract class SpecWidget`?